azurerm/resource_arm_app_service_test.go

```diff
@@ -2666,3 +3054,552 @@ resource "azurerm_app_service" "test" {
 }
 `, rInt, location, rInt, rInt)
 }
+
+func testAccAzureRMAppService_authSettingsAdditionalLoginParams(rInt int, location string, tenantID string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_app_service_plan" "test" {
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  name                = "acctestRG-%d"
+  kind                = "Linux"
```

Comment:
because this is provisioning a Linux App Service Plan this also needs the `reserved` field set to `true`